run-on-lido/stvaults/pdg.md

```diff
@@ -53,106 +53,141 @@ Advantages:
+
    - Method called: `Dashboard.setPDGPolicy(PDGPolicy)`.
-   - Caller must have the `DEFAULT_ADMIN_ROLE` role.
+   - Caller must have the `DEFAULT_ADMIN_ROLE` role (or be its admin).
 
    PDG Policies:
-      - `STRICT` (Default): deposits require the full PDG process.
-      - `ALLOW_PROVE`: allows the Node Operator to prove unknown validators to PDG.
-      - `ALLOW_DEPOSIT_AND_PROVE`: allows the Node Operator to perform unguaranteed deposits (bypassing the predeposit requirement) and prove unknown validators.
 
-3. The Node Operator assigns roles `NODE_OPERATOR_UNGUARANTEED_DEPOSIT_ROLE` and `NODE_OPERATOR_PROVE_UNKNOWN_VALIDATOR_ROLE`. In the current guide, the Node Operator assigns these roles to the same address as `Depositor`, but this is unnecessary.
+   - `STRICT` (Default): deposits require the full PDG process.
+   - `ALLOW_PROVE`: allows proving unknown validators to PDG (but not unguaranteed deposits).
+   - `ALLOW_DEPOSIT_AND_PROVE`: allows both unguaranteed deposits (bypassing the predeposit requirement) and proving unknown validators.
+
+3. The Node Operator Manager grants the necessary roles to the address that will perform unguaranteed deposits and proving. In this guide, we assume both roles are granted to the same address referred to as `Depositor`.
+
+   - Method called: `Dashboard.grantRole(role, account)` for each role.
+   - Caller must have the `NODE_OPERATOR_MANAGER_ROLE` role.
+   - Roles to grant:
+     - `NODE_OPERATOR_UNGUARANTEED_DEPOSIT_ROLE` — allows calling `unguaranteedDepositToBeaconChain()`
+     - `NODE_OPERATOR_PROVE_UNKNOWN_VALIDATOR_ROLE` — allows calling `proveUnknownValidatorsToPDG()`
+
+4. **(Optional)** The Node Operator assigns a `Depositor` address in the PDG contract that will be authorized to top up validators via PDG after they are proven.
 
-4. The Node Operator _optionally_ assigns a `Depositor` address that will deposit and top up validators with ETH from the stVault balance. 
    - Method called: `PredepositGuarantee.setNodeOperatorDepositor(newDepositor)`.
-   - Caller must have the `NODE_OPERATOR_ROLE` role.
-5. The `Depositor` generates validator keys and predeposit data.
-6. The `Depositor` deposits 32 ETH (minimum 32 ETH) from the vault balance to the validator via the Dashboard contract.
+   - Caller must be the Node Operator address (as registered in the StakingVault via `nodeOperator()`).
+   - Note: This step is only required if there are plans to use `PredepositGuarantee.topUpExistingValidators()` later. The unguaranteed deposit flow does not require this.
+
+5. The `Depositor` generates validator keys and deposit data with:
+
+   - Withdrawal credentials in `0x02-type` format pointing to the stVault address.
+   - Deposit amount up to 2048 ETH.
+
+6. The `Depositor` deposits 32 ETH (for example) from the vault balance to the validator via the Dashboard contract.
```

Comment:
Maybe emphesise that this is example to showcase topup prooving, and for 2048 eth deposits no need to grant NODE_OPERATOR_PROVE_UNKNOWN_VALIDATOR_ROLE ?